repokid/utils/roledata.py

```diff
@@ -279,6 +283,46 @@ def update_aardvark_data(aardvark_data, roles):
             LOGGER.error('Dynamo table error: {}'.format(e))
 
 
+def update_no_repo_permissions(role, newly_added_permissions):
+    """
+    Update Dyanmo entry for newly added permissions. Any that were newly detected get added with an expiration
+    date of now plus the config setting for 'repo_requirements': 'exclude_new_permissions_for_days'. Expired entries
+    get deleted. Also update the role object with the new no-repo-permissions.
+
+    Args:
+        role
+        newly_added_permissions (set)
+
+    Returns:
+        None
+    """
+    current_ignored_permissions = _get_role_data(role.role_id, fields=['NoRepoPermissions']).get(
+                                                 'NoRepoPermissions', {})
+    new_ignored_permissions = {}
+
+    current_time = int(time.time())
+    new_perms_expire_time = current_time + (
+        24 * 60 * 60 * repokid.repokid.CONFIG['repo_requirements']['exclude_new_permissions_for_days'])
```

Comment:
If someone generated a config before `exclude_new_permissions_for_days` was an option, and then upgraded to this codebase, how would repokid behave?